### PR TITLE
basic redux setup

### PR DESCRIPTION
**Pull request by lcurvelo27**
Sorry, it took me longer than expected to set this up.

### CODE REVIEW

**Review by jfirwin — APPROVED** [review]
This looks great! :shipit:

File: front-end/src/components/search.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
 import React, {Component} from 'react'
 import axios from 'axios'
+import { connect } from 'react-redux'
+import action from '../ducks/action'
+const {updateName} = action
```
Comment (lukeschunk) [inline front-end/src/components/search.js:5]: one thing i would say in the future you can do:
```
import { updateName } from '../ducks/action'
```

Comment (jfirwin) [reply front-end/src/components/search.js:5]: true, good catch.

Comment (lcurvelo27) [reply front-end/src/components/search.js:5]: Will do from now on. Thank you!

**Pull request merged**